### DISCUSSION TITLE
Approved for GitHub Student Developer Pack but Haven't Received Academic Benefits

### DISCUSSION THREAD

**Discussion by AKomplished-bug** [community/community#133125] [GitHub Education] [answered]
### Hi everyone,

I got my application for the GitHub Student Developer Pack approved three days ago. However, I have not yet received any of the academic benefits associated with the pack.

Is there a potential issue with my application that I might not be aware of? Any guidance or assistance would be greatly appreciated.

Thank you,
Athul

**Comment by davevad93** [comment, marked as answer]
Hi @AKomplished-bug ,

It usually takes a few days, but sometimes it can take a little longer, even a couple of weeks. The waiting time may vary depending on different factors, such as the volume of requests.

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @AKomplished-bug was approved and your benefits are now available.`

For further reference you can look at these discussions: #132790 , #131665 , #132470
